docs/sources/review/lint-prose/dictionary/add-words.md

```diff
@@ -0,0 +1,242 @@
+---
+description: How to add a word to the Grafana Labs dictionary template.
+menuTitle: Add words
+title: Add words to the Grafana Labs dictionary
+---
+
+# Add words to the Grafana Labs dictionary
+
+The Grafana Labs documentation team maintains a dictionary used for spell checking.
+The same dictionary is used to generate some Vale rules from the metadata in the word definition.
+
+The template file uses the Jsonnet programming language but you don't need to know Jsonnet to add a new word.
+
+This topic explains how to perform the following tasks:
+
+- [Add a verb](#add-a-verb)
+- [Add a product](#add-a-product)
+- [Add an abbreviation](#add-an-abbreviation)
+
+For more complicated words, if you're comfortable with writing Jsonnet, refer to [word metadata reference](https://grafana.com/docs/writers-toolkit/review/lint-prose/dictionary/#word-metadata.
+If you're not comfortable writing Jsonnet, [create an issue](https://github.com/grafana/writers-toolkit/issues/new?title=Grafana.Spelling%20%3A%20%3CWORD%3E), and a maintainer can add it for you.
+
+## Before you begin
+
+- Clone the [Writers' Toolkit repository](https://github.com/grafana/writers-toolkit).
+
+  For more information, refer to [Create a local repository](https://grafana.com/docs/writers-toolkit/write/tooling-and-workflows/#create-a-local-repository).
+
+- Create a branch for your change.
+
+  For more information, refer to [Create a branch from the default remote branch](https://grafana.com/docs/writers-toolkit/write/tooling-and-workflows/#create-a-branch-from-the-default-remote-branch).
+
+## Add a verb
+
```

Comment:
Add words to the dictionary to ensure consistent and comprehensive coverage of words within our documentation. You can add a verb, noun, adverb, or adjective. To add product names or abbreviations, see those specific guidelines that follow.